lib/help.txt

```diff
@@ -19,6 +19,11 @@ Usage: elasticdump --input SOURCE --output DESTINATION [OPTIONS]
                     Delete documents one-by-one from the input as they are 
                     moved.  Will not delete the source index
                     (default: false)
+
+--update
+                    Update existing documents if they exist in the destination.
+                    See "op_type", and / or bulk operation "create" vs "index".
+                    (default: true)
```

Comment:
Add a note here about what will happen when false, ie: "when false, existing documents will be skipped"